x-pack/plugins/gis/public/components/widget_overlay/layer_control/layer_toc/toc_entry/index.js

```diff
@@ -6,9 +6,9 @@
 
 import _ from 'lodash';
 import { connect } from 'react-redux';
-import { TOCEntry } from './toc_entry';
+import { TOCEntry } from './view';
```

Comment:
Why are you renaming this to `view`? It is a lot more difficult in the debugger and IDE when every file is called `view`. Why not name the view files so there is less name collision?

Reply:
consistency with namings for our other components that use an index file

Reply:
But why not go the other way, and rename `view.js` to `somethingThatActuallyMeansSomething.js`? Seems like reverting to `view.js` goes backwards